src/query/service/tests/it/storages/testdata/settings_table.txt

```diff
@@ -15,6 +15,7 @@ DB.Table: 'system'.'settings', Table: settings-table_id:1, ver:0, Engine: System
 | 'enable_hive_parquet_predict_pushdown'         | '1'            | '1'            | 'SESSION' | 'Enable hive parquet predict pushdown  by setting this variable to 1, default value: 1'                                                                                               | 'UInt64' |
 | 'enable_query_result_cache'                    | '0'            | '0'            | 'SESSION' | 'Enables caching query results to improve performance for identical queries.'                                                                                                         | 'UInt64' |
 | 'enable_recluster_after_write'                 | '1'            | '1'            | 'SESSION' | 'Enables re-clustering after write(copy/replace-into).'                                                                                                                               | 'UInt64' |
+| 'enable_refresh_aggregating_index_after_write' | '0'            | '0'            | 'SESSION' | 'Refresh aggregating index after new data written'                                                                                                                                    | 'UInt64' |
```

Comment:
If the user create a sync agg index, but disable this setting, and insert data. Agg index blocks belonging to this data will never be created?

Reply:
Yep, this setting is just for now and will be enabled default when all parts(copy/replace into) are finished.